.github/workflows/build.yml

```diff
@@ -0,0 +1,101 @@
+name: Build All
+
+on: [push]
+
+jobs:
+
+  build_scala2_10:
```

Comment:
Why not use a build matrix strategy instead of 4 copies of the same job?

Reply:
That would be great if I can use it. I need `2.10` for caching and `2.10.7` for sbt build. Do you know if I can have pairs in matrix?

Reply:
Actually I have an idea. I'll test it with my personal project first. Thanks for your suggestion to use matrix.

Reply:
It looks good.
https://github.com/Kevin-Lee/scala-hedgehog/commit/553054fdc20cdaaedc79060ae5a294b9588bf84e/checks